core/cache/watcher.go

```diff
@@ -248,15 +248,6 @@ type PredicateStringsWatcher struct {
 	fn predicateFunc
 }
 
-// newChangeWatcher provides a PredicateStringsWatcher which notifies
-// with all strings passed to it.
-func newChangeWatcher(values ...string) *PredicateStringsWatcher {
```

Comment:
Dead code.

Reply:
I left it on purpose.  Hoping it'd be used shortly. oh well

Reply:
I'll restore it.